infra/gcp/ensure-release-projects.sh

```diff
@@ -58,6 +59,8 @@ for PROJECT; do
     color 6 "Ensuring project exists: ${PROJECT}"
     ensure_project "${PROJECT}"
 
+    continue
```

Comment:
Does this not mean everything below it is never run?

Reply:
🤦 good catch